docs/notebooks/measurement-error-mitigation.ipynb

```diff
@@ -0,0 +1,546 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "remove_cell"
+    ]
+   },
+   "source": [
+    "# Measurement Error Mitigation on Quantum Inspire\n",
+    "Pieter Eendebak <pieter.eendebak@tno.nl><br/>\n",
+    "\n",
+    "For more details see [Qiskit textbook on measurement error mitigation](https://qiskit.org/textbook/ch-quantum-hardware/measurement-error-mitigation.html) and [Mitiq documentation](https://mitiq.readthedocs.io/en/stable/readme.html).\n",
+    "\n",
+    "For quantum devices, in particular spin-qubits, the measurement error is significant with respect to other sources of errors. We can reduce the effect of measurement errors using measurement error mitigation. This is particularly important when different qubits have different readout fidelities or when there is a cross-talk between the qubits readouts.\n",
+    "\n",
+    "To install the required dependencies for this notebook run: `pip install qiskit rich quantuminspire requests`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import logging\n",
+    "\n",
+    "import numpy as np\n",
+    "import qiskit\n",
+    "import requests\n",
+    "from qiskit import Aer, QuantumCircuit, QuantumRegister, assemble, execute, transpile\n",
+    "from qiskit.ignis.mitigation.measurement import CompleteMeasFitter, complete_meas_cal\n",
+    "from qiskit.providers.aer.noise import NoiseModel, ReadoutError\n",
+    "from qiskit.providers.aer.noise.errors import depolarizing_error, pauli_error\n",
+    "from qiskit.visualization import plot_histogram\n",
+    "from quantuminspire.api import QuantumInspireAPI\n",
+    "from quantuminspire.credentials import get_authentication\n",
```

Comment:
For this import to work you need the latest release of quantuminspire 1.5.0 (`pip install quantuminspire --upgrade`). It was released yesterday. That's also the reason I do the review now and not earlier.